Lib/multiprocessing/managers.py

```diff
@@ -1159,15 +1159,19 @@ def __imul__(self, value):
         self._callmethod('__imul__', (value,))
         return self
 
+    __class_getitem__ = classmethod(types.GenericAlias)
+
 
-DictProxy = MakeProxyType('DictProxy', (
+BaseDictProxy = MakeProxyType('DictProxy', (
```

Comment:
Would it be possible to make it private? I mean, just add an underscore prefix: _BaseDictProxy.